README.md

```diff
@@ -278,23 +324,22 @@ public function deleteData(int employeeID) returns (json) {
     json updateStatus = {};
 
     string sqlString = "DELETE FROM EMPLOYEES WHERE EmployeeID = ?";
-    // Delete existing data by invoking update action defined in ballerina sql client
+    // Delete existing data by invoking update remote function defined in ballerina sql client
     var ret = employeeDB->update(sqlString, employeeID);
-    match ret {
-        int updateRowCount => {
-            updateStatus = { "Status": "Data Deleted Successfully" };
-        }
-        error err => {
-            updateStatus = { "Status": "Data Not Deleted", "Error": err.message };
-        }
+    if (ret is int) {
+        updateStatus = { "Status": "Data Deleted Successfully" };
+    } else if (ret is error) {
+        updateStatus = { "Status": "Data Not Deleted",  "Error": "Error occurred during delete operation" };
+        // Log the error for the service maintainers.
+        log:printError("Error occurred during delete operation", err = ret);
     }
     return updateStatus;
 }
 ```
 
-The `endpoint` keyword in Ballerina refers to a connection with a remote service. In this case, the remote service is a MySQL database. `employeeDB` is the reference name for the SQL endpoint. The rest of the code is for preparing SQL queries and executing them by calling the `update` action in the `ballerina/mysql` module.
+A remote function in Ballerina indicates that it will be communicating with some remote service through the network. In this case, the remote service is a MySQL database. `employeeDB` is the reference name for the MySQL client object which encapsulates aforementioned set of remote functions. The rest of the code is for preparing SQL queries and executing them by calling these remote functions of the Ballerina MySQL client.
```

Comment:
Shall we use present tense instead of future since that is the practice we follow in docs. `that it communicates`